modules/tiles/src/tileset/helpers/bounding-volume.js

```diff
@@ -9,7 +9,7 @@ import {assert} from '@loaders.gl/loader-utils';
 
 // const scratchProjectedBoundingSphere = new BoundingSphere();
 
-const defined = x => x !== undefined;
+const defined = x => x !== undefined && x !== null;
```

Comment:
is this used very frequently, if so make it traditional `function` instead of `arrow function`, for perf. reasons.